obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -0,0 +1,30 @@
+---
+- name: 'Parse tito.props'
+  tito_props_find_package:
+    package: "{{ inventory_hostname }}"
+    tito_props_path: "{{ tito_props_path }}"
+  register: tito_props
```

Comment:
What about if I am either not using tito or I am not configuring tito in this way?

Reply:
does obal support build environments where tito isn't used?

Reply:
Fair point to the first question, but what about the second?

Reply:
I ask because whitelist is specific to koji only (copr and brew releasers don't use it I think).

Reply:
tito cannot operate without a props file

```
$ cd foreman-packaging
$ rm -rf rel-eng
$ cd packages/katello/rubygem-hammer_cli_katello/
$ tito build --srpm --offline --dist .el7 --scm tfm
Creating output directory: /tmp/tito
ERROR: Unable to locate branch configuration: /home/adprice/workspace/foreman-packaging/rel-eng/tito.props
Please run 'tito init'
```

Reply:
Yea, its not the props file itself, sorry for the confusion. It's the contents of the props file. Some of what is in the pops file is releaser specific. I believe the part you are relying on here is Koji releaser specific.

Reply:
Looking at `man tito.props`, I'm seeing that `whitelist` (as well as `disttag`) are used by copr and koji (which would include brew, correct?).

Reply:
BrewReleaser is different than KojiReleaser because of dist-git. You can specify disttag per tag in koji which is independent. But whitelist is not supported by Brew unless this has changed in recent tito. I am still curious what necessitates the complexity over just running ""tito build --srpm"" ?

Reply:
> I am still curious what necessitates the complexity over just running ""tito build --srpm"" ?

I guess I just assumed those flags were necessary, otherwise why do they exist? But If they aren't needed then that can greatly  reduce the complexity of this task.

Reply:
The `--offline` tag was needed previously when using tito tagging to manage packages. It was a way to tell tito "do not reach out to git and confirm this tag exists" for pre-release testing. For scratch builds in Koji, I believe it does not matter if you set --scl properly on the SRPM it only matters when doing a release to koji.